tutor/jobs.py

```diff
@@ -144,7 +144,17 @@ def set_theme(theme_name: str, domain_names: List[str], runner: BaseJobRunner) -
 def get_all_openedx_domains(config: Config) -> List[str]:
     return [
         get_typed(config, "LMS_HOST", str),
+        get_typed(config, "LMS_HOST", str)
+        + ":"
+        + get_typed(config, "NGINX_HTTP_PORT", str),
```

Comment:
I don't understand. Why do we need to append the nginx port here? When running a separate web proxy, we are not supposed to access the lms without passing through this proxy first.

Reply:
For the case like https://discuss.overhang.io/t/install-themes-not-working-even-indigo/2140 ? There's no harm adding this line I think?

Reply:
If `NGINX_HTTP_PORT` is different than 80 (say: 81), then the above config will only work for http://lmshost:81. Yet, users are really not supposed to open the LMS from this URL. For instance, this host is not part of the Django ALLOWED_HOSTS.

So I think we must drop the NGINX_HTTP_PORT part here.